keras/layers/containers.py

```diff
@@ -75,6 +75,19 @@ def updates(self):
         return updates
 
     @property
+    def state_updates(self):
+        state_updates = []
+        for l in self.layers:
+            if l.trainable and getattr(l, 'stateful', False):
```

Comment:
To tell you the truth, I wasn't 100% sure what `trainable` was signaling and it seemed to indicate that a layer had been compiled and would accept data so it felt right...  I took it out though.